libafl_bolts/src/lib.rs

```diff
@@ -1064,7 +1081,7 @@ pub fn get_thread_id() -> u64 {
 #[allow(clippy::cast_sign_loss)]
 /// Return thread ID without using TLS
 pub fn get_thread_id() -> u64 {
-    use libc::{SYS_gettid, syscall};
+    use libc::{syscall, SYS_gettid};
```

Comment:
is this a cargo fmt update of some sorts?

Reply:
yes

Reply:
Seems like your fmt is not the same as the CI one